test-e2e/push-golden.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
```

Comment:
@tylerferrara -- To preserve the history of this file, let's do the following:
- One commit to `git mv` the file up a directory
- A second commit that adds the logic for the auditor pushes

Reply:
Absolutely! This was the original plan, though I'm still testing the script to make sure it works with the auditor. This PR will only be modifying `push-golden.sh` to keep the diff small.

Reply:
@tylerferrara -- this still needs to be done as separate commits using `git mv` to preserve the history and make the changes to content clear in the diff

Reply:
After changing more than (the default similarity), git interprets this as a (creation and deletion) rather than a mv.

Reply:
@tylerferrara -- This is why it needs to be done as two separate commits. It'll look like a creation/deletion in GitHub's UI, but git's tracking of the file should remain intact.

Reply:
Even separating the `git mv` to another commit still shows the same creation/deletion.

Reply:
I spoke offline with @tylerferrara. There will be 1 commit doing a pure `git mv` only and subsequent commits actually making changes to the business logic.

Reply:
As it stands now it appears this has been done already. Thanks @tylerferrara !

Reply:
Looks good now. Thanks @tylerferrara!
(FYI, you'll be able to see the difference if you step through commit-by-commit; the GitHub UI does not handle moves well in the aggregated diff view.)